Test/main.cpp

```diff
@@ -0,0 +1,113 @@
+#include <iostream>
+#include <fstream>
+
+
+ //input = std::cin , output = std::cout; Can this be done?
```

Comment:
You can put `using std::cin;` and `using std::cout;` up here and then just write cin and cout below.